.devcontainer/post-create-command.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Install Guard CLI
+curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/aws-cloudformation/cloudformation-guard/main/install-guard.sh | sh
```

Comment:
While I have no issue with this why isn't this baked into the container?

Reply:
Not a feature in the devcontainer.json, we are dockerfileless around here